.github/workflows/nfr.yml

```diff
@@ -35,104 +38,142 @@ permissions:
   contents: read
 
 jobs:
+  vars:
+    name: Set up vars
+    runs-on: ubuntu-22.04
+    outputs:
+      test_label: ${{ github.event.inputs.test_label || 'all' }}
+      version: ${{ github.event.inputs.version || 'edge' }}
+      image_tag: ${{ github.event.inputs.image_tag || 'edge' }}
+      types: ${{ steps.var.outputs.types }}
+    permissions:
+      contents: read
+    steps:
+      - name: Set vars
+        id: var
+        run: |
+          if ${{ github.event.inputs.type == 'both' || github.event_name == 'schedule' }}; then
+          echo 'types=["oss","plus"]' >> $GITHUB_OUTPUT
+          else
+          echo 'types=["${{ github.event.inputs.type }}"]' >> $GITHUB_OUTPUT
+          fi
+
   setup-and-run-tests:
     name: Setup and Run NFR Tests
     runs-on: ubuntu-22.04
     permissions:
-      contents: write # needed for opening PR with the results files
-      pull-requests: write # needed for opening PR with the results files
+      contents: read
       id-token: write # needed for authenticating to GCP
+    needs: vars
+    strategy:
+      matrix:
```

Comment:
So this will result in two PRs being opened from the one pipeline, right?

Reply:
just one PR with all the results